addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1851,6 +1852,7 @@ const Wysiwyg = Widget.extend({
         // snippet is a media.
         const isInMedia = $target.is(mediaSelector) && !$target.parent().hasClass('o_stars') && e.target &&
             (e.target.isContentEditable || (e.target.parentElement && e.target.parentElement.isContentEditable));
+            // TODO also change condition here ? If so, maybe create a function ?
```

Comment:
Please do :) 

We often do that in fact. We notice something in the code and we think "that cannot be right, I'm pretty sure it will break if I managed to do X" then we proceed to do X and sometimes we were right and there is a bug there and sometimes there isn't because there is another mechanism in play that we didn't necessarily think of when reading the code.

Reply:
So, I do see a bug now with icons in the s_company_team: the "Replace" button does not appear (anymore, because it definitely did at some point).
Investigating if this `IsInMedia` boolean has anything to do with it, I think the answer is no. But I haven't found the culprit either. Maybe the explanation is in the docs (italics added)
> Update any editor UI _that is not handled by the editor itself_.
Could you help point me where the editor itself handles the UI changes related to the type of selected content ? Specifically what code decides if "Replace" media button appears.

Separately, I notice a weird code snippet later in this method
```python
if (isInMedia) { // do stuff }
if (isInMedia) { // do more stuff }
```
Any reason for such an oddity ?

Reply:
Just to be clear, right now the s_company_team icons only have the "replace" button appear if I manually add the `contenteditable=True` attribute to them. The social media icons appear as they should (custom media has the "replace" button, standard ones don't).
Changing the condition for `isInMedia` doesn't change any of those behaviors.

Reply:
> Could you help point me where the editor itself handles the UI changes related to the type of selected content ? Specifically what code decides if "Replace" media button appears.

I feel like we're talking about `_updateToolbar` ?

> Any reason for such an oddity ?

Yep. The second condition used to be different that the first one. Whoever changed the second one did not realize that the second one was now equivalent to the first one and therefore was not needed anymore. I had a look using `tig blame` and it looks to be me, in fact: 35cc6049a8bcc3c80cb4eb2468f85d09734814b8

Reply:
Closing since `isInMedia` seems unrelated to the task.